go/pkg/client/exec.go

```diff
@@ -192,16 +194,13 @@ func buildCommand(ac *Action) *repb.Command {
 		Arguments: ac.Args,
 		// Do not use OutputFiles and OutputDirs directly from the Action, as we need to sort them which
 		// implies modification.
-		OutputFiles:       make([]string, len(ac.OutputFiles)),
-		OutputDirectories: make([]string, len(ac.OutputDirs)),
+		OutputPaths: append(ac.OutputFiles, ac.OutputDirs...),
 		Platform: &repb.Platform{
 			Properties: []*repb.Platform_Property{{Name: containerImagePropertyName, Value: ac.DockerImage}},
 		},
 	}
-	copy(cmd.OutputFiles, ac.OutputFiles)
-	copy(cmd.OutputDirectories, ac.OutputDirs)
-	sort.Strings(cmd.OutputFiles)
-	sort.Strings(cmd.OutputDirectories)
+	sort.Strings(cmd.OutputPaths)
```

Comment:
Do we also need to maintain the old fields for older servers and implementations?